frontend/templates/product-list/sections/FilterableProductsSection/RefinementMultiSelect.tsx

```diff
@@ -68,28 +68,31 @@ const MultiSelectItem = React.memo(function MultiSelectItem({
 }: MultiSelectItemProps) {
    const [isRefined, setIsRefined] = useDecoupledState(item.isRefined);
 
-   return (
-      <HStack key={item.label} justify="space-between">
-         <Checkbox
-            role="option"
-            value={item.value}
-            isChecked={isRefined}
-            onChange={() => {
-               setIsRefined((current) => !current);
-               refine(item.value);
-            }}
-            data-value={item.value}
-         >
-            {item.label}
-         </Checkbox>
-         <Text
-            size="sm"
-            fontFamily="sans-serif"
-            color="gray.500"
-            fontWeight={isRefined ? 'bold' : 'inherit'}
-         >
-            {item.count}
-         </Text>
-      </HStack>
-   );
+   if (item.count > 0) {
```

Comment:
Let's get rid of these ifs since we should be operating under the assumption that nothing should show up with item.count = 0 and this will hide it if that breaks.

Reply:
Resolved d8e2412efef6e68936c1fd6462b0c5b09fccfafd